sdcm/cluster_k8s/__init__.py

```diff
@@ -119,6 +126,7 @@ def start_minikube(node: cluster.BaseNode) -> None:
         minikube_start_script = dedent(f"""
             sysctl fs.protected_regular=0
             minikube start --driver=none --extra-config=apiserver.service-node-port-range=1-65535
+            ip link set docker0 promisc on
```

Comment:
Is it required for upgrade test?

Reply:
This one is required to having truncate work

Reply:
Will it be set back at the end of the test?

Reply:
@amoskong no need to set it back since we are running on minikube instance that is destroyed at the end of the test.